client/components/domains/featured-domain-suggestions/featured-domain-suggestion/index.jsx

```diff
@@ -0,0 +1,129 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	shouldBundleDomainWithPlan,
+	getDomainPriceRule,
+	hasDomainInCart,
+} from 'lib/cart-values/cart-items';
+import DomainSuggestion from 'components/domains/domain-suggestion';
+import ProgressBar from 'components/progress-bar';
+
+const NOTICE_GREEN = '#4ab866';
+
+export class FeaturedDomainSuggestion extends Component {
```

Comment:
I lack the knowledge of how React works but is there a way to reuse/expand/inherit the `DomainRegistrationSuggestion` component and override only the render part? Or maybe just extend the `DomainRegistrationSuggestion` to have a separate property that would make it `featured`? I'm not 100% sure what's the best approach here, it just seems that those two components are very similar. Or maybe I'm wrong?

Reply:
Let me address this comment as well as your concerns mentioned in the main body of your review in a separate comment below.